Makefile

```diff
@@ -409,13 +413,17 @@ geth-connect: ##@other Connect to Geth on the device
 	build/bin/geth attach http://localhost:8545
 
 ios-clean: SHELL := /bin/sh
+ios-clean: ios-simulator-cache-clean
 ios-clean: ##@prepare Clean iOS build artifacts
 	git clean -dxf -f target/ios
 
-android-clean: export TARGET := gradle
+android-clean: SHELL := /bin/sh
 android-clean: ##@prepare Clean Gradle state
 	git clean -dxf -f ./android/app/build; \
-	[[ -d android/.gradle ]] && cd android && ./gradlew clean
+	rm -rf android/.gradle \
+	rm -rf android/build \
+	rm -rf ~/.gradle
```

Comment:
You know this could just be a single `rm` right?

Also both `android/.gradle` and `android/build` could have been handled by the `git clean` here by adding more paths.

Reply:
Nice! 
I wasn't aware about the `git clean` approach. it does seem neater.
Thanks for sharing this. I'll fix this in a follow up PR.